### ISSUE TITLE
relabel "guides" to "Best practices" and move under "Using Notify" primary nav

### ISSUE THREAD

**Issue by dmvancura** [GSA/notifications-admin#2234]
with new "Using Notify" content going live (likely) by 1/9/25 we can keep the Best practices content as it was in tests, but relabel it as such i.e., get rid of the "Guides" section label.

and "Best practices" section/label is moved under the "Using Notify" content section

may require, or may already have, the site nav and side nav outlined in wireframe

**Comment by dmvancura** [comment]
oh @heyitsmebev should i update this ticket to get "Guidance" removed or hidden from the site? it's in prep to update the whole Using Notify section (aside from the wonderfully built Best practices section)

**Comment by heyitsmebev** [comment]
@dmvancura I believe this ticket has been completed by @alexjanousekGSA in https://github.com/GSA/notifications-admin/issues/2233

**Comment by ecayer** [comment]
Hi friends! I'm clearly a bit confused 😅 The thing that says "Guidance" is also linked as "personalize messages" from the Get Started page... so this is presumably a different "Guidance" from thing that became Best Practices! But "Guidance" still doesn't seem like the right title for the page that's actually about formatting and personalizing?

**Comment by dmvancura** [comment]
correct, that is a legacy label. I'd like it to say "How to" temporarily. Does that work? @ecayer Because page content is going to go through the content revamp with the rest of the Using Notify content (aside from Best practices content)

**Comment by ecayer** [comment]
Works for me, thank you!

**Comment by heyitsmebev** [comment]
![Image](https://github.com/user-attachments/assets/dd7f7068-973b-446a-96ad-737889505b72)
